integrations/datadog/overrides.py

```diff
@@ -12,14 +12,19 @@
 
 
 class SLOHistorySelector(Selector):
-    sample_interval_period_in_days: int = Field(
-        alias="sampleIntervalPeriodInDays", default=7
-    )
+    timeframe: int = Field(alias="timeframe", default=7)
+    period_of_time_in_years: int = Field(alias="periodOfTimeInYears", default=1)
 
-    @validator("sample_interval_period_in_days")
-    def validate_resource_kinds_min_size(cls, v: int) -> int:
+    @validator("timeframe")
+    def validate_timeframe_field(cls, v: int) -> int:
         if v < 1:
-            raise ValueError("sampleIntervalPeriodInDays must be greater than 0")
+            raise ValueError("timeframe must be greater than 0")
+        return v
+
+    @validator("period_of_time_in_years")
+    def validate_period_of_time_in_years(cls, v: int) -> int:
+        if v > 1:
+            raise ValueError("period_of_time_in_years must be less than 1")
```

Comment:
not sure i understand, firstly the message should be less or equal to 1, and secondly - it defined as int so the value could be only 1